### PR TITLE
Document that File.cd!/2 affects the BEAM globally (#10695)

### PR DESCRIPTION
**Pull request by hrubi**

### CODE REVIEW

File: lib/elixir/lib/file.ex [hunk 1]
```diff
@@ -1542,6 +1542,12 @@ defmodule File do
   executes the given function and then reverts back
   to the previous path regardless of whether there is an exception.
 
+  The current working directory is temporarilly set for the BEAM globally. This
```
Comment (whatyouhide) [inline lib/elixir/lib/file.ex:1545]:
```suggestion
  The current working directory is temporarily set for the BEAM globally. This
```

**Review by whatyouhide — APPROVED** [review]

File: lib/elixir/lib/file.ex (hunk 1 above)
Comment (whatyouhide) [reply lib/elixir/lib/file.ex:1545]: Ah, too late!

Comment (josevalim) [reply lib/elixir/lib/file.ex:1545]: It is a copy and paste of File.cd!, so better to fix both separately.

Comment (hrubi) [reply lib/elixir/lib/file.ex:1545]: Well, it was a copy paste, but I've added this one word.. and messed up. Here's the fix https://github.com/elixir-lang/elixir/pull/10697

**Pull request merged**